src/test/java/com/softserveinc/ita/dkrutenko/RozetkaTests.java

```diff
@@ -38,16 +43,18 @@ public void verifyShoppingCartTest(String brandNameItem, String requiredItem) {
         //fill search with some brand name, click search button and click on required product
         searchGoods.fillSearchField(brandNameItem);
         searchGoods.clickSearchButton();
-        searchGoods.findActualtem(requiredItem);
+        ShoppingCartPage shoppingCartPage = searchGoods.findActualtem(requiredItem);
```

Comment:
if are using var's, you should use it also for such variables var shoppingCartPage

Reply:
done